bench-scripts/apache_bench.sh

```diff
@@ -891,6 +929,8 @@ function setup_tests {
 	cd "${WORKSPACE_ROOT}"
 	clean_build
 
+	install_haproxy
```

Comment:
here we call install_haproxy with no arguments. it feels like we install/build ha-proxy with openssl which comes from master branch. this makes me wonder how we run tests for HA proxy bult with other SSL versions.

Reply:
I added to other branches too. The `run_tests` will need refactoring as it is just code repetition, but I'd like to do that in the next PR to keep this only about haproxy